src/View/Helper/AbstractTranslatorHelper.php

```diff
@@ -28,7 +28,7 @@
      *
      * @var string
      */
-    protected $translatorTextDomain = 'default';
+    protected $translatorTextDomain = null;
```

Comment:
Yes, but being this a class designed for inheritance, it makes the BC break very noticeable